crates/project/src/lsp_store.rs

```diff
@@ -6370,7 +6370,6 @@ impl LspStore {
                     .await
                     .into_iter()
                     .flat_map(|(_, actions)| actions)
-                    .filter(|help| !help.label.is_empty())
```

Comment:
I reviewed the code, but it doesn't seem necessary to deliberately filter empty `label`s here.
While there's a possibility that an invalid language server might send such `label`s, even in those cases, this PR's implementation won't cause a panic and just render empty `label` as is.